pandas/tests/dtypes/test_common.py

```diff
@@ -706,3 +707,24 @@ def test__get_dtype_fails(input_param, expected_error_message):
 )
 def test__is_dtype_type(input_param, result):
     assert com._is_dtype_type(input_param, lambda tipo: tipo == result)
+
+
+@pytest.mark.parametrize("from_type", [np.datetime64, np.timedelta64])
+@pytest.mark.parametrize(
```

Comment:
Could this get used elsewhere? If so might be good to make a shared fixture for `numeric_dtype` or something to the effect

Reply:
This particular setup might be a little specific in that all the `to_types` have lesser precision than the `from_types`, but did you have something in mind?

Reply:
this is ok